ovn-tester/ovn_workload.py

```diff
@@ -610,3 +628,7 @@ def select_worker_for_port(self):
         self.last_selected_worker += 1
         self.last_selected_worker %= len(self.worker_nodes)
         return self.worker_nodes[self.last_selected_worker]
+
+    def provision_ns_lb(self, lb):
```

Comment:
I think this should be renamed to `provision_lb` or `provision_load_balancer`. There is nothing about this method that is inherent to Namespaces.